projects/laji/src/app/+observation/observation-data.service.ts

```diff
@@ -1,26 +1,16 @@
-import {gql} from 'apollo-angular';
 import { Injectable } from '@angular/core';
 import { Observable, of } from 'rxjs';
-import { map, startWith } from 'rxjs/operators';
+import { map, shareReplay, startWith } from 'rxjs/operators';
 
-import { DocumentNode } from 'graphql';
-import { GraphQLService } from '../graph-ql/service/graph-ql.service';
-import { HistoryService } from '../shared/service/history.service';
 import { WarehouseQueryInterface } from '../shared/model/WarehouseQueryInterface';
 import { SearchQueryService } from './search-query.service';
 import { WarehouseApi } from '../shared/api/WarehouseApi';
 import { PlatformService } from '../root/platform.service';
 
 export interface IObservationData {
-  units: {
-    total: number;
-  };
-  species: {
-    total: number;
-  };
-  private: {
-    total: number;
-  };
+  count: number;
+  speciesCount: number;
+  securedCount: number;
```

Comment:
Good job on simplifying the datamodel. Could be get a more descriptive name for `count` as well? Count of what, `unitsCount`?

Reply:
Thanks. One more thing - apparently these can also be `null`. Could you add that to the typings? That should increase safety. I wonder how the compiler doesn't complaing about setting these to `null` with the `empty()` method... :thinking:

Reply:
Well, never mind, that's out of scope of this PR since it was already that way.